src/meeting/teams/speakersObserver.ts

```
@@ -0,0 +1,522 @@
+            ) {
+                //live platform: Handle live platform - EXACT SAME AS EXTENSION
+                const name =
+                    element.getAttribute('aria-label')?.split(',')[0] || ''
+                console.log(`[TEAMS-DEBUG] Live platform - found name: "${name}"`)
+                if (name) {
+                                    // Only process if we have a name
+                                    const micIcon = element.querySelector(
+                                        '[data-cid="roster-participant-muted"]',
+                                    )
+                                    const isMuted = micIcon ? true : false
+                                    const voiceLevelIndicator = element.querySelector(
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)